include/nbl/ui/CWindowWin32.h

```diff
@@ -42,9 +43,52 @@ class CWindowWin32 final : public IWindowWin32
 
 	~CWindowWin32() override;
 private:
-	CWindowWin32(core::smart_refctd_ptr<system::ISystem>&& sys, int _x, int _y, uint32_t _w, uint32_t _h, E_CREATE_FLAGS _flags);
+	CWindowWin32(core::smart_refctd_ptr<system::ISystem>&& sys, int32_t _x, int32_t _y, uint32_t _w, uint32_t _h, E_CREATE_FLAGS _flags);
 
     native_handle_t m_native;
+
+	std::map<HANDLE, core::smart_refctd_ptr<IMouseEventChannel>> m_mouseEventChannel;
```

Comment:
use `nbl::core::` containers instead

Reply:
btw what is HANDLE? cant see anything like it in win api

Reply:
> btw what is HANDLE? cant see anything like it in win api

Those are descriptors for many things in winapi, just a `void*` typedef

Reply:
afaik all things that have `H` prefix in their types are handles as well (like HWND), i guess they made specific typedefs for the most commonly used things and left `HANDLE` for stuff which is used a bit less.